yew/src/app.rs

```diff
@@ -39,6 +39,13 @@ where
     /// will render the model to a virtual DOM tree. If you would like to pass props,
     /// use the `mount_with_props` method.
     pub fn mount(self, element: Element) -> ComponentLink<COMP> {
+        #[cfg(feature = "dev")]
+        let _ = wasm_bindgen_futures::spawn_local(async move {
+            let new_debugger = crate::dev::DebuggerConnection::new().await;
```

Comment:
I don't think this is necessary if you have a message queue. The debugger connection message queue should be immediately available

Reply:
No – it's here from before I added the message queue.